sunbird_service_api_test/src/test/java/org/sunbird/integration/test/common/BaseCitrusTestRunner.java

```diff
@@ -86,10 +86,12 @@ public void performPostTest(
                 requestJson,
                 contentType,
                 TestActionUtil.getHeaders(isAuthRequired)));
-    runner.http(
-        builder ->
-            TestActionUtil.getResponseTestAction(
-                builder, LMS_ENDPOINT, templateDir, testName, responseCode, responseJson));
+    if (!responseCode.equals(HttpStatus.BAD_REQUEST)) {
```

Comment:
Have null check on responseJson and in the null case call the overloaded method which doesn't check response.